.pre-commit-config.yaml

```diff
@@ -0,0 +1 @@
+/nix/store/0qn5cx859h6b2hdpll99qzb39j7lvfc6-pre-commit-config.json
```

Comment:
issue: accidental symlink addition?
This should probably go in .gitignore. I don't know why it wasn't already in there.

Reply:
true, let me fix that